scss/styles.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
```

Comment:
Perfect, only one comment for comings prs, normally we use hex convention for colors.

Reply:
Nice! I will use hex, thank you :smile: